store/iavl/store.go

```diff
@@ -279,7 +279,13 @@ func (st *Store) Export(version int64) (*iavl.Exporter, error) {
 	if !ok || tree == nil {
 		return nil, fmt.Errorf("iavl export failed: unable to fetch tree for version %v", version)
 	}
-	return tree.Export(), nil
```

Comment:
export should return error imo. Something to change in the near future.

Reply:
Yes, would you like me to try to return an error from deep within the stack?  Checking nil is not the most ideal right?

Reply:
I think it will require at least some agreement from the iavl maintainer.  Do you know who that is?

Reply:
This is the location of the related PR I've opened where we could return an explicit error:
https://github.com/cosmos/iavl/pull/622/files#diff-9d9ca52939a19f26358e2b5b35ca938679b381be35f4c54379df94c4fa4d1233R37

Reply:
But it would require iavl to tag a new release before cosmos-sdk would be able to compile.